images/linux/scripts/installers/containers.sh

```diff
@@ -12,3 +13,8 @@ apt-get update -qq
 apt-get -qq -y install podman buildah skopeo
 mkdir -p /etc/containers
 echo -e "[registries.search]\nregistries = ['docker.io', 'quay.io']" | tee /etc/containers/registries.conf
+
+# Check that tools are installed properly
+podman --version
+buildah --version
+skopeo --version
```

Comment:
I suggest changing it to another approach, which is implemented in many other scripts
```
if ! podman --version; then
    echo "Podman was not installed"
    exit 1
fi
```

Reply:
Thank you for the suggestion, I've changed these lines.